modules/social_features/social_core/social_core.install

```diff
@@ -531,3 +530,10 @@ function social_core_update_130007(): void {
   // Change file_managed table only files from field_file field.
   \Drupal::database()->query("UPDATE {file_managed} SET {file_managed}.uri = REPLACE({file_managed}.uri, 'secret://', 'private://') WHERE fid IN (SELECT {node__field_files}.field_files_target_id FROM {node__field_files})");
 }
+
+/**
+ * Replace permission administer taxonomy with access taxonomy overview for SM.
```

Comment:
@Macsym2018 Please replace comment with `Revoke administer taxonomy permission for sitemanager`

Reply:
@ribel Changed